internal/transport/grpcframer/errors.go

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package grpcframer
+
+// ErrorCode represents an HTTP/2 Error Code.
+// See https://httpwg.org/specs/rfc7540.html#ErrorCodes.
+type ErrorCode uint32
```

Comment:
Seems like we are missing the stringer method on this type. Would be nice to limit behavior changes when we migrate to use the new types